### PR TITLE
Add 1.12 images

### PR DESCRIPTION
**Pull request by VitorTrin**

### CODE REVIEW

**Comment by felicianotech** [comment]
Hi. Thank you for this PR unfortunately we're going to go with the one [I made](https://github.com/CircleCI-Public/cimg-elixir/pull/54). The main reason being that this PR wasn't created using the build scripts as explained in the readme. It ensures that the pieces that need to be updated are when publishing new image tags.

Thank you.

**Comment by VitorTrin** [comment]
As long as a 1.12 image is avaible I don't care who commits.

**Comment by tjmcs** [comment]
Hi @VitorTrin; apologies for the delay in dealing with this pull request, we've had a team member out on vacation and have been trying to resolve the differences between your pull request and the pull request that he created to replace this one just before he left.

I'm going to reopen this pull request in favor of his, and we'd like to ask that you make a slight change to this pull request so that it uses a slightly older base image version.  If you can make those changes (two lines in your PR that I'll comment on separately) we can get this pull request merged quickly.

Thanks again for making this contribution to our repository, and hopefully we can get this sorted out soon (as soon as your changes to this PR are in, I hope).  We appreciate your patience :)

File: 1.12/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# vim:set ft=dockerfile:
+
+FROM cimg/base:2021.05
```
Comment (tjmcs) [inline 1.12/Dockerfile:3]: Can you down-rev this base image version to 2021.04?  We tend to only update the base version occasionally (and in unison across our convenience images) in order to ensure that these base images are cached properly.

File: Dockerfile.template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # vim:set ft=dockerfile:
 
-FROM cimg/%%PARENT%%:2021.03
+FROM cimg/%%PARENT%%:2021.05
```
Comment (tjmcs) [inline Dockerfile.template:3]: Can you down-rev this base image version to `2021.04`?  We tend to only update the base version occasionally (and in unison across our convenience images) in order to ensure that these base images are cached properly.

**Review by KyleTryon — APPROVED** [review]

**Comment by KyleTryon** [comment]
Thank you again @VitorTrin and sorry for the delay. We highly appreciate your help. Shipping now!
large ship

**Comment by KyleTryon** [comment]
https://hub.docker.com/r/cimg/elixir/tags?page=1&ordering=last_updated&name=1.12
https://circleci.com/developer/images/image/cimg/elixir (will appear here within 15 minutes)

**Pull request merged**